### ISSUE TITLE
Random Seed Settings for all Sampling and Initialization calls

### ISSUE THREAD

**Issue by nilmeier** [CODAIT/r4ml#48]
It would be nice to be able to set random seeds for all of the places where random number generators are used so that we can have reproduceable (to numerical precision) results for our tests.

**Comment by aloknsingh** [comment]
Yes this is there , please create relevant jira also in our internal system
